Clarify the object id required for Cosmos RBAC auth (#95278)
Azure/azure-content: articles/cosmos-db/how-to-setup-rbac.md

$resourceGroupName = "<myResourceGroup>"
$accountName = "<myCosmosAccount>"
$readOnlyRoleDefinitionId = "<roleDefinitionId>" # as fetched above

> # Make sure to use the Object ID as found in the Enterprise applications section of the Azure Active Directory portal blade.
> # the other Object ID does not work for this

Author: ```suggestion
# For Service Principals make sure to use the Object ID as found in the Enterprise applications section of the Azure Active Directory portal blade.
```

$resourceGroupName = "<myResourceGroup>"
$accountName = "<myCosmosAccount>"
$readOnlyRoleDefinitionId = "<roleDefinitionId>" # as fetched above
# Make sure to use the Object ID as found in the Enterprise applications section of the Azure Active Directory portal blade.
# the other Object ID does not work for this

> # for managed identities make sure you update the filter in the portal to include them by default it's just 'Enterprise Applications'

Author: ```suggestion
```

resourceGroupName='<myResourceGroup>'
accountName='<myCosmosAccount>'
readOnlyRoleDefinitionId = '<roleDefinitionId>' # as fetched above

> # Make sure to use the Object ID as found in the Enterprise applications section of the Azure Active Directory portal blade.
> # the other Object ID does not work for this

Author: ```suggestion
# For Service Principals make sure to use the Object ID as found in the Enterprise applications section of the Azure Active Directory portal blade.
```

resourceGroupName='<myResourceGroup>'
accountName='<myCosmosAccount>'
readOnlyRoleDefinitionId = '<roleDefinitionId>' # as fetched above
# Make sure to use the Object ID as found in the Enterprise applications section of the Azure Active Directory portal blade.
# the other Object ID does not work for this

> # for managed identities make sure you update the filter in the portal to include them by default it's just 'Enterprise Applications'

Author: ```suggestion
```
